HW3.js

```diff
@@ -0,0 +1,189 @@
+//#1
+
+console.log(addThemAll(2,4)); // 6
+console.log(addThemAll(1,2,3,4)); // 10
+console.log(addThemAll(5,5,10)); // 20
+
+function addThemAll(...args) {
+    let sum = 0;
+
+    for (let arg of args) {
+        sum += arg;
+    }
+
+    return sum;
+}
+
+//#2
+
+console.log(multiply(5)(5))		// 25
+console.log(multiply(2)(-2))	// -4
+console.log(multiply(4)(3))		// 12
+
+function multiply(a) {
+    let product = a;
+
+    return function (b) {
+        return product *= b;
+    }
```

Comment:
```suggestion
function multiply(a) {
    return function (b) {
        return a *= b;
    }
```

Об'являти змінну, що використовується в одному місці, не має сенсу, можна спростити

Reply:
Або взагалі - `const multiply = (a) => (b) => a *= b;`

Але це вже як приклад.